index.js

```diff
@@ -119,3 +119,22 @@ const portfolioCard = (pro) => {
   btndiv.append(btn);
 };
 portfolio.forEach((pro) => portfolioCard(pro));
+
+function validate() {
+  const contactForm = document.getElementById('contactForm');
+  const email = document.getElementById('email').value;
+  const text = document.getElementById('text');
+  const pattern = /^[a-z]\w+@[a-z_]+?\.[a-z]{2,3}$/;
+  if (email.match(pattern)) {
+    contactForm.classList.add('valid');
+    contactForm.classList.remove('invalid');
+    text.innerHTML = 'Your email is valid';
+    text.style.color = '#00ff00';
+  } else {
+    contactForm.classList.remove('valid');
+    contactForm.classList.add('invalid');
+    text.innerHTML = 'Your email is invalid';
+    text.style.color = '#ff0000';
```

Comment:
- [ ] Great job by implementing the error handling message for email. But, for now, there is a simple issue you have to fix to go for the next project. For now, once we open your website, immediately we see the error message: `Your email is invalid` Please, kindly hide it, so that the user can see it just once he fills the email form with characters that don't match your pattern. This could improve your UX.